specification/appendix/commitment_discounts.md

```diff
@@ -0,0 +1,274 @@
+# Commitment Discounts
+
+A commitment discount is a billing discount that offers reduced rates on preselected SKUs in exchange for an obligated usage or spend amount over a predefined term. Commitment discounts typically consist of a set of purchase and usage records within cost and usage datasets.
+
+## Purchasing
+
+While customers are bound to the term of a commitment discount, cloud-service providers, or CSPs, offer various payment options before and/or during the term:
+
+* *Upfront* - The commitment discount is paid in full before the term begins
+* *Monthly* - The commitment discount is paid monthly over the term.
```

Comment:
Is `Monthly` the right term here?  There may be other grains we want to include here.  Suggest using `Recurring` or similar instead.

Reply:
Recurring works.